### ISSUE TITLE
New installation object created when migrating from ObjC SDK

### ISSUE THREAD

**Issue by mtrezza** [parse-community/Parse-Swift#426]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-Swift/security/policy).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/main/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/Parse-Swift/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse Swift SDK](https://github.com/parse-community/Parse-Swift/releases). 

### Issue Description

After migrating an app to the Parse Swift SDK, launching the app will create a new `_Installation` object with a new `installationId`. It will appear as if the app had been uninstalled and then reinstalled, even though it was only updated with the new Parse Swift SDK.

### Steps to reproduce
n/a

### Actual Outcome

A new installation object is created.

### Expected Outcome

Any existing installation object should be used as the current installation.

### Environment

Client
- Parse Swift SDK version: `4.14.2`

**Comment by cbaker6** [comment]
Those developers seeing this issue should note that this is not true and is not an issue. If you really want to migrate from the Objective-C SDK to the Swift SDK, see the convo, sample app, and a developer who successfully migrated [here](https://community.parseplatform.org/t/how-to-transition-from-the-old-parse-framework/2807/19?u=cbaker6).

Since I’ve already solved this problem, I will not be spending any additional time on Objective-C SDK migration, so those looking for a better way will need to depend on someone else to provide a different solution.

**Comment by mtrezza** [comment]
If this is not an issue anymore, can we remove it from the migration guide?

**Comment by mtrezza** [comment]
I've searched through code for instructions on how to migrate the installation:

```swift
let objcInstallation = PFInstallation.current()!
let swiftInstallation = try await Installation.become(objcInstallation.objectId!)
```

It seems to expect that the Parse ObjC SDK is installed in parallel to the Parse Swift SDK, is that correct?

**Comment by cbaker6** [comment]
> It seems to expect that the Parse ObjC SDK is installed in parallel to the Parse Swift SDK, is that correct?

The description of https://github.com/parse-community/Parse-Swift/pull/407#issue-1368984914 states:

> Adding a method like this allows developers to use their own ways to get a particular objectId and use it to migrate to the Swift SDK.

As for:
> is that even recommended and tested or could they interfere with each other?

Why would they interfere with each other? These are two different SDK's and they only thing they have in common is transmitting data to/from a Parse Server. The probability of these SDK's interfering with each other is the same as using any to other frameworks in the same project. The sample app linked imports and configures the SDK's in the same [file](https://github.com/netreconlab/ParseMigrateKeychain/blob/1c44700fe056d55445a470217c0460e080f1457c/ParseMigrateKeychain/ContentViewModel.swift#L9-L10). Any conflicts between the SDK's will be those created by the developer.

**Comment by mtrezza** [comment]
Thanks for confirming that there is no interference. These answers may seem obvious to you, but these are questions that developers may have when migrating.

> it doesn't matter if a developer has both SDK's in their app or if they have saved the objectId for their installation somewhere else

It sounds like we have no easy approach? Having both SDKs in 2 project certainly looks odd and is not ideal from a resource standpoint. Storing the objectId somewhere else again requires an app update to be adopted as the basis for migration, also not ideal.

**Comment by mtrezza** [comment]
I don't mind changing this to "feature", we just need to track the issue.
